apps/mempool/docker-compose.yml

```diff
@@ -13,9 +13,9 @@ services:
     logging: *default-logging
     restart: on-failure
     stop_grace_period: 1m
-    command: "./wait-for db:3306 --timeout=720 -- nginx -g 'daemon off;'"
+    command: /bin/sh -c "cp /etc/nginx/nginx.conf /patch/nginx.conf && sed -i 's/8080/${APP_MEMPOOL_PORT}/g' /patch/nginx.conf && ./wait-for db:3306 --timeout=720 -- nginx -g 'daemon off;' -c /patch/nginx.conf"
```

Comment:
This should be done in the mempool Docker image, not from our side.

This still implicitly relies on the port being hardcoded to 8080 in the mempool source.

Also I'd recommend using `envsubst` not hand rolling it with `sed`. If the NGINX config is ever changed to include the string `8080` elsehwere, (a hash?) this sed script will break the configuration file.

Reply:
Okay sure but if this is the only issue I would prefer to fix it in a future release

Reply:
I've made a PR to handle this case and the MySQL vars!

Reply:
This is fixed in the last version of Mempool (v2.1.1) 
Thanks to Wiz :rocket: